doc/zh-cn/plugins/hmac-auth.md

```diff
@@ -142,6 +143,21 @@ Accept-Ranges: bytes
 ```
 
 
+## 自定义 header 名称
+
+我们可以在 `conf/config.yaml` 中，`plugin_attr` 下添加插件的属性配置来自定义参数 header 名称。
+
+```yaml
+plugin_attr:
+  hmac-auth:
+    signature_key: X-APISIX-HMAC-SIGNATURE
+    algorithm_key: X-APISIX-HMAC-ALGORITHM
+    http_date_key: X-APISIX-DATE
+    access_key: X-APISIX-HMAC-ACCESS-KEY
+    signed_headers_key: X-APISIX-HMAC-SIGNED-HEADERS
+```
+
```

Comment:
After setting, how to use it, need an example.

Reply:
added.